apps/component-examples/examples/payout-transactions-list.js

```diff
@@ -0,0 +1,97 @@
+require('dotenv').config({ path: '../../.env' });
+
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3000;
+const authTokenEndpoint = process.env.AUTH_TOKEN_ENDPOINT;
+const webComponentTokenEndpoint = process.env.WEB_COMPONENT_TOKEN_ENDPOINT;
+const subAccountId = process.env.SUB_ACCOUNT_ID;
+const payoutId = process.env.PAYOUT_ID;
+const clientId = process.env.CLIENT_ID;
+const clientSecret = process.env.CLIENT_SECRET;
+
+app.use(
+  '/scripts',
+  express.static(__dirname + '/../node_modules/@justifi/webcomponents/dist/')
+);
+app.use('/styles', express.static(__dirname + '/../css/'));
+
+async function getToken() {
+  const requestBody = JSON.stringify({
+    client_id: clientId,
+    client_secret: clientSecret,
+  });
+
+  let response;
+  try {
+    response = await fetch(authTokenEndpoint, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: requestBody,
+    });
+  } catch (error) {
+    console.log('ERROR:', error);
+  }
+
+  const { access_token } = await response.json();
+  return access_token;
+}
+
+async function getWebComponentToken(token) {
+  const response = await fetch(webComponentTokenEndpoint, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${token}`,
+    },
+    body: JSON.stringify({
+      resources: [`read:payments:${subAccountId}`],
+    }),
+  });
+
+  const { access_token } = await response.json();
+  return access_token;
+}
+
+app.get('/', async (req, res) => {
+  const token = await getToken();
+  const webComponentToken = await getWebComponentToken(token);
+
+  res.send(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>JustiFi Payout Transactions List Component</title>
+        <script type="module" src="/scripts/webcomponents/webcomponents.esm.js"></script>
+        <link rel="stylesheet" href="/styles/theme.css">
+        <link rel="stylesheet" href="/styles/example.css">
+      </head>
+      <body>
+        <div class="list-component-wrapper">
+          <justifi-payout-transactions-list
+            payout-id="${payoutId}"
+            auth-token="${webComponentToken}"
+            api-origin=
```

Comment:
Should this line be remove or completed?

Reply:
Removed it - thanks!